src/pages/Index.vue

```diff
@@ -65,12 +67,15 @@
             <div class="tile is-parent">
               <article class="tile is-child box home-content">
                 <p class="title">The Lab</p>
-                <p class="subtitle">Located at 171 MLK Student Union</p>
+                <p class="subtitle">
+                  Located at 171 MLK Student Union
+                  <g-link to="/docs/services/lab"> See more » </g-link>
+                </p>
                 <div class="content">
                   <p>
                     Find out how to
-                    <a href="https://goo.gl/maps/USohudX21j5H1JQC7">get there</a
-                    >. Just show your Cal ID at the door!
+                    <a href="/docs/services/lab/#location">get there</a>. Just
```

Comment:
Could you convert this to a `g-link` instead of an `a` so that the client-side routing kicks in?

Reply:
Sorry, just one more thing. The `href` also needs to be changed to `to` because `g-link` is a pain in the ass

Reply:
my bad, should've noticed it sooner

Reply:
No worries, thanks for fixing